convert2rhel/actions/post_conversion/modified_rpm_files_diff.py

```diff
@@ -46,7 +46,7 @@ def run(self):
             self.add_message(
                 level="INFO",
                 id="SKIPPED_MODIFIED_RPM_FILES_DIFF",
-                title="Skipped comparison of 'rpm -Va' output from before and after the conversion.",
+                title="Skipped comparison of 'rpm -Va' output from before and after the conversion",
```

Comment:
Dot removed here as well

Reply:
This is intentional. It's one of the goals of the PR - to remove full stops from message titles for consistency. Most titles didn't have it, so I opted for removing them from all.

The message titles are printed this way:
```
(INFO) MODIFIED_RPM_FILES_DIFF::FOUND_MODIFIED_RPM_FILES - Modified rpm files from before and after the conversion were found
     Description: ...
```

In Insights they are displayed in this fashion:
![Screenshot from 2024-09-03 17-20-22](https://github.com/user-attachments/assets/ebbb69be-d420-42f6-a4b4-4d59d7e83901)

In both cases (CLI, Insights) it's more suitable to have the titles without full stops.